src/main/java/co/rsk/federate/BtcToRskClient.java

```diff
@@ -155,30 +154,31 @@ public synchronized Map<Sha256Hash, List<Proof>> getTransactionsToSendToRsk() {
 
     public void updateBridge() {
         try {
-            if (federation == null) {
-                logger.warn("updateBridge skipped because no Federation is associated to this BtcToRskClient");
-            }
-            if (!nodeBlockProcessor.hasBetterBlockToSync()) {
-                logger.debug("Updating bridge");
-                int numberOfBlocksSent = updateBridgeBtcBlockchain();
-                logger.debug("Updated bridge blockchain with {} blocks", numberOfBlocksSent);
-                logger.debug("Updating transactions and sending update");
-                updateBridgeBtcCoinbaseTransactions();
-                updateBridgeBtcTransactions();
-                federatorSupport.sendUpdateCollections();
-            } else {
-                logger.warn("updateBridge skipped because the node is syncing blocks");
+            if (federation == null || nodeBlockProcessor.hasBetterBlockToSync()) {
```

Comment:
Not a big deal, but I would rather have two `if` with each check separated. 

For me, the implementation below would be easier to understand at first glance.
```
if (federation == null){
logger.warn("[updateBridge] updateBridge skipped because {}", reason);
return;
}

if (nodeBlockProcessor.hasBetterBlockToSync()){
logger.warn("[updateBridge] updateBridge skipped because {}", reason);
return;
}
```

Reply:
Ok, that makes sense. Updated, thanks